src/baseInterceptor.js

```diff
@@ -34,122 +35,563 @@ baseInterceptor.prototype.getColorName = function(colArgs) {
-];
+const htmlColors = [{
```

Comment:
@spiray  - due to the length of this variable - would it make sense to move this variable into a separate file?

Reply:
I think so, however, I think that should be a separate PR as this file has many "helper" functions that we should really modularize. We should really create a `restructure` issue for each file and then fix the linting errors separately.